Qt.py

```diff
@@ -893,9 +893,14 @@ def load(self, uifile, *args, **kwargs):
                 etree = ElementTree()
                 etree.parse(uifile)
 
-                return Qt._QtUiTools.QUiLoader.load(
+                widget = Qt._QtUiTools.QUiLoader.load(
                     self, uifile, *args, **kwargs)
 
+                # Prevent RuntimeError in Nuke 9.0v9 with PySide 1.0.9
+                Qt.QtCompat._loadUi_temp = widget.parentWidget()
```

Comment:
Ah, didn't catch that. No, that should take care of it.

In that case I'm less keen on having Nuke specific code in Qt.py. If we could find a way of reproducing the bug in isolation and resolve that, that would be ideal.